scripts/helper-update-version

```diff
@@ -0,0 +1,17 @@
+ #!/bin/bash
```

Comment:
I would not put the script into the `scripts` folder as it is made for the application itself.
I would add this script at the root of this repository.

Reply:
Done